R/population-workflow.R

```diff
@@ -292,3 +312,21 @@ PopulationWorkflow <- R6::R6Class(
     }
   )
 )
+
+
+#' @export
+PopulationWorkflowTypes <- enum(c(
+  "pediatric",
+  "parallelComparison",
+  "ratioComparison"
+))
+
+#' @export
+DemographyPaths <- enum(c(

```

Comment:
This is already available in ospsuite-r
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/c6dea52b9edb7509182fa67316cd9ae481d3b453/R/standard-path.R#L18

If something is missing, let's add. But let's not duplicate